### ISSUE TITLE
Different grammars for different Python versions

### ISSUE THREAD

**Issue by pavelvelikhov** [pavelvelikhov/pythonql#9]
Currently PythonQL does support different Python versions for execution.
However, internally Python 3.3 grammar is used, so if you write statements that don't
conform to Python 3.3 grammar, PythonQL will report a syntax error.

We should be able to support multiple versions of Python in PythonQL in the future.

**Comment by SowmyaLR** [comment]
> Currently PythonQL does support different Python versions for execution.
> However, internally Python 3.3 grammar is used, so if you write statements that don't
> conform to Python 3.3 grammar, PythonQL will report a syntax error.
> 
> We should be able to support multiple versions of Python in PythonQL in the future.

Yes. When I tried pythonql in python-3.8 and jython-2.7 the following code 
`print([ select y for x in [1,2,3] let y = x**2 ])`

It showed output as
**SyntaxError: invalid syntax**
